include/dgl/array.h

```diff
@@ -164,12 +164,12 @@ struct COOMatrix {
 ///////////////////////// CSR routines //////////////////////////
 
 /*! \brief Return true if the value (row, col) is non-zero */
-bool CSRIsNonZero(CSRMatrix , int64_t row, int64_t col);
+bool CSRIsNonZero(CSRMatrix , int64_t row, int64_t col, bool sorted);
```

Comment:
The sorted flag could be directly in `CSRMatrix` class so no need to add the argument to the functions.